src/components/Navbar/Navbar.component.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext, useState } from 'react';
+import { StateContext } from '../../utils/State';
+import { NavBar, InputStyled, SwitchStyled, LoginButton, BaseButton, InputContainer, LoginContainer, DarkModeStyled } from './Navbar.styled';
+import { LoginOutlined, LogoutOutlined } from '@ant-design/icons';
+import { Tooltip } from 'antd';
+import Login from './Login'
+import { useHistory } from 'react-router';
+
+
+const Navbar = () => {
+
+    const { DarkMode, setDarkMode, Sesion, setSesion, SearchVideo, setSearchVideo, 
+        Theme: { SecondaryColor, TextColor } } = useContext(StateContext);
+    const [SearchVideoNav, setSearchVideoNav] = useState(SearchVideo)
+    const [OpenModal, setOpenModal] = useState("");
+    const history = useHistory();
+
+    const onChangeSesion = () => {
+        if(Sesion){
+            setSesion(null);
+        } else {
+            setOpenModal("Login");
+        }
+    }
+
+    const Modals = {
```

Comment:
I understand the purpose for this object but I think that this is only adding more code and it isn't necessary for your application, I suggest to remove it otherwise you could have some issues when implementing your UT also I suggest to break your component in smaller components and with that your code'll look cleaner and it'll be easier to write your tests.